contracts/base/Loans.sol

```diff
@@ -361,4 +361,15 @@ contract Loans is LoansInterface, SignerRole {
             liquidated: false
         });
     }
+
+    function isCollateralSentEnough(
+        uint256 msgValue,
+        uint256 amountToBorrow,
+        uint256 collateralRatio
+    ) internal view returns (bool) {
+        uint256 oneLendingTokenPriceWeis = uint256(priceOracle.getLatestAnswer());
+        uint256 collateralSent = msgValue.div(oneLendingTokenPriceWeis);
+        uint256 collateralNeeded = amountToBorrow.mul(collateralRatio).div(1000);
```

Comment:
this should be `.div(10000);`
the contract has a constant `TEN_THOUSAND` for this purpose. This is due to the fact that collateral ratio is a percentage with 2 decimals. So a collateralRatio of 65.43% is stored as 6543. Therefore to find the collateral needed you must do `amountToBorrow * 0.6543`. Which is `amountToBorrow.mul(collateralRatio).div(10000);`

Reply:
Okay, perfect. I was using 3 digits for the collateral ratio. I have changed it.